examples/credit_loan_decisions.py

```diff
@@ -0,0 +1,1359 @@
+)
+
+# %%
+# __ThresholdOptimizer__
+
+# %%
+metricframe_inprocess.overall[metrics_to_report]
+
+# %%
+create_metricframe_w_errors(
+    metricframe_postprocess, metrics_to_report, metric_error_pairs
+)
+
+# %%
+metricframe_postprocess.overall[metrics_to_report]
+
+# %%
+# We see both of our fairness-aware models yield a slight decrease in the
+# *balanced_accuracy* for *male applicants* compared to our fairness-unaware
+# model. In the *reductions* model, we see a decrease in the *false positive
+# rate* for *female applicants*. This is accompanied by an increase in the
+# *false negative rate* for *male applicants*. However, overall, the *equalized
+# odds difference* for the *reductions* models is lower than that of the
+# original fairness-unaware model.
+
+
+# %%
+# ## Conclusion and Discussion
+# ============================
+
+# %%
+# In this case study, we walked through the process of assessing a credit
+# decision model for gender-related performance disparities. Our analysis
+# follows closely the work done in the `*Microsoft/EY white paper*
+# <https://www.microsoft.com/en-us/research/uploads/prod/2020/09/Fairlearn-EY_WhitePaper-2020-09-22.pdf>`_
+# where they used the *Fairlearn* toolkit to perform an audit of a
+# fairness-unaware tree-based model. We applied a *post-processing* and
+# *reductions* mitigation techniques to mitigate the *equalized odds difference*
+# in our model.
+#
+# Through the *reductions* process, we generated a model that reduces the
+# *equalized odds difference* of the original model without a drastic increase
+# in the *balanced error score*. If this were a real model being developed a
+# financial institution, the *balanced error score* would be a proxy for the
+# profitability of the model. By maintaining a relatively similar *balanced
+# error score*, we've produced a model that preserves profitability to the firm
+# while producing more fair and equitable outcomes for women in this scenario.
+
+# %%
+# Designing a Model Card
+# --------------------------
+
+# %%
+# A key facet of Responsible Machine Learning is responsible documentation
+# practices. `Mitchell et al. (2019) <https://arxiv.org/abs/1810.03993>`_
+# proposed the model card framework for documentating and reporting model
+# training details and deployment considerations.
+#
+# A *model card* contains sections for documenting training and evaluation
+# dataset descriptions, ethical concerns, and quantitative evaluation summaries.
+# In practice, we would ideally create a model card for our model before
+# deploying it in production. Although we will not be producing a model card in
+# this case study, interested readers can learn more about creating model cards
+# using the *Model Card Toolkit* from the `Fairlearn PyCon 2022 tutorial
+# <https://colab.research.google.com/github/LeJit/talks/blob/pycon_2022/2022_pycon/pycon-2022-students.ipynb>`_.
+
+# %%
+# Fairness Under Unawareness
```

Comment:
```suggestion
# Fairness under unawareness
```

Reply:
_Fairness Under Unawareness_ (with that capitalization structure) is the literal name of the paper I'm referencing in this section .